java/statestore/src/main/java/sleeper/statestore/dynamodb/DynamoDBFileInfoStore.java

```diff
@@ -210,18 +208,25 @@ public void atomicallyUpdateJobStatusOfFiles(String jobId, List<FileInfo> files)
     }
 
     @Override
-    public void deleteReadyForGCFile(String filename) throws StateStoreException {
+    public void deleteReadyForGCFiles(List<String> filenames) throws StateStoreException {
+        TransactWriteItemsRequest transactWriteItemsRequest = new TransactWriteItemsRequest()
+                .withTransactItems(filenames.stream()
```

Comment:
This will fail if there are more than 100 files in the filenames list. We need to batch up the list into groups of <100 files.